data/hai/mercenaries.txt

```diff
@@ -0,0 +1,828 @@
+	names "pirate"
+	personality
+		disables plunders harvests
+	variant 2
+		"Marauder Firebird (Engines)"
+	variant 3
+		"Marauder Firebird"
+	variant 3
+		"Marauder Arrow (Weapons)"
+		"Marauder Bounder"
+	variant 3
+		"Marauder Falcon"
+	variant 3
+		"Marauder Raven (Weapons)"
+	variant
+		"Marauder Raven"
+		"Marauder Quicksilver (Weapons)" 2
+	variant 3
+		"Marauder Splinter (Weapons)"
+	variant 2
+		"Marauder Manta" 2
+	variant
+		"Marauder Leviathan (Engines)"
+	variant
+		"Marauder Raven" 2
+	variant
+		"Sea Scorpion (Ion)"
+	variant
+		"Sea Scorpion (Pulse)"
+	variant
+		"Lightning Bug" 2
+
+
+
+phrase "mercenaries defeated" 
+	word
+		`After disabling the last of the pirate ships, you wonder what could have motivated such an attack. Do you possess something valuable to them? When they entered the system, they only sent one message: "nemo me impune lacessit". It is probably best to store whatever they were interested in on a planet (using U) or get rid of it.`
+
+# attacks to get the bars back, in Hai and human space; as they have more influence in human space they are more numerous there
+mission "Mercenaries: Nemo Me Impune Lacessit"
+	minor
+	landing
+	invisible
+	repeat
+	source
+		near "Ya Hai" 10
+	to offer
+		has "outfit: Gold Bars"
+		not "mercenaries: sold bars"
+		not "mercenaries: took out leader"
+		random < 3
+	npc disable
+		government "Mercenaries"
+		system
+			distance 2 5
+		fleet "Marauders" 3
+		dialog phrase "mercenaries defeated"
+	to fail
+		or
+			has "mercenaries: took out leader"
+			not "outfit: Gold Bars"
+
+mission "Mercenaries: Nemo Me Impune Lacessit 2"
+	minor
+	landing
+	invisible
+	repeat
+	source
+		near Sol 100
+	to offer
+		"outfit: Gold Bars" > 15
+		not "mercenaries: sold bars"
+		not "mercenaries: took out leader"
+		random < 1
+	npc disable
+		government "Mercenaries"
+		system
+			distance 2 5
+		fleet "Marauders" 5
+		dialog phrase "mercenaries defeated"
+	to fail
+		or
+			has "mercenaries: took out leader"
+			not "outfit: Gold Bars"
+
+
+
+# SPOILER: mercenaries sometimes save pirates from slavery, in exchange for a life debt.
+# Some of your crew may be part of that, or maybe it is just any pirate from this world that tries to get them, attracted by the huge reward.
+mission "Mercenaries: Betrayal For The Bars"
+	landing
+	invisible
+	source
+		government "Pirate"
+	to offer
+		random < 40
+		has "Unfettered: Jump Drive Retrieval: done"
+		has "flagship crew" > 4
+		"outfit (flagship installed): Gold Bars" + "outfit (flagship cargo): Gold Bars" >= 5
+	on offer
+		conversation
+			`As you land on the pirate world, you head for the hatch, preparing to enter the spaceport, but passing by the cargo bay you think you hear a sound. Odd... There shouldn't be anyone there right now, knowing your crew's habits, as most of them have already disembarked.`
+			choice
+				`(Its nothing. Probably just a rat.)`
+				`(Go investigate.)`
+					goto investigate
+			action
+				outfit "Gold Bars" -5
+			``
```

Comment:
What is this for? Can't you just defer in `choice`?

Reply:
I need to take the bars first, and `` is needed to have an endpoint